src/docs/asciidoc/reference/manage/environment-add.adoc

```diff
@@ -0,0 +1,4 @@
+:relative-path: ../../../
+include::{docdir}/variables.adoc[]
+
+NOTE: The *environment* support is not completely done. It will be soon. The management of this environments will be done via the *web console*.
```

Comment:
Use convention for notes and tips
```
.Title of the note (here it could be Contact us for adding new environments)
[NOTE]
====
content of the note
====
```

We should also explain that currently, ZetaPush can manage environments for our clients